exchanges/bitstamp/bitstamp_test.go

```diff
@@ -965,29 +974,45 @@ func TestOrderbookZeroBidPrice(t *testing.T) {
 
 func TestGetWithdrawalsHistory(t *testing.T) {
 	t.Parallel()
+
 	sharedtestvalues.SkipTestIfCredentialsUnset(t, b)
-	_, err := b.GetWithdrawalsHistory(context.Background(), currency.BTC, asset.Spot)
-	if err != nil {
-		t.Error(err)
+	h, err := b.GetWithdrawalsHistory(context.Background(), currency.BTC, asset.Spot)
+	assert.NoError(t, err, "GetWithdrawalsHistory should not error")
+	if mockTests {
+		assert.NotEmpty(t, h, "GetWithdrawalRequests should return a withdrawal request")
+		if mockTests {
+			assert.NotEmpty(t, h, "WithdrawalHistory should not be empty")
+			for _, req := range h {
+				assert.Equal(t, time.Date(2022, time.January, 31, 16, 7, 32, 0, time.UTC), req.Timestamp, "Timestamp should match")
+				assert.Equal(t, "BTC", req.Currency, "Currency should match")
+				assert.Equal(t, 0.00006000, req.Amount, "Amount should match")
+			}
+		}
 	}
 }
-
 func TestGetOrderInfo(t *testing.T) {
 	t.Parallel()
+
 	sharedtestvalues.SkipTestIfCredentialsUnset(t, b)
-	_, err := b.GetOrderInfo(context.Background(), "1234", currency.NewPair(currency.BTC, currency.USD), asset.Spot)
-	if err != nil {
-		t.Error(err)
+	o, err := b.GetOrderInfo(context.Background(), "1458532827766784", currency.NewPair(currency.BTC, currency.USD), asset.Spot)
+	if mockTests {
+		assert.NoError(t, err, "GetOrderInfo should not error")
+		assert.Equal(t, time.Date(2022, time.January, 31, 14, 43, 15, 0, time.UTC), o.Date, "order date should match")
+		assert.Equal(t, "1458532827766784", o.OrderID, "order ID should match")
+		assert.Equal(t, 200.00, o.Amount, "amount should match")
+		assert.Equal(t, 50.00, o.Price, "price should match")
+	} else {
+		assert.ErrorContains(t, err, "authenticated request failed Order not found")
 	}
 }
 
 func TestFetchWSAuth(t *testing.T) {
 	t.Parallel()
+
 	resp, err := b.FetchWSAuth(context.TODO())
```

Comment:
Don't know why this wasn't there before, but please add

```go
if !mocktests {
    sharedtestvalues.SkipTestIfCredentialsUnset(t, b)
}
```

Reply:
added